src/core/pointcloud/qgspointcloudlayer.cpp

```diff
@@ -422,6 +422,51 @@ QString QgsPointCloudLayer::htmlMetadata() const
                 + ( pointCount < 0 ? tr( "unknown" ) : locale.toString( static_cast<qlonglong>( pointCount ) ) )
                 + QStringLiteral( "</td></tr>\n" );
 
+  const QVariantMap originalMetadata = mDataProvider ? mDataProvider->originalMetadata() : QVariantMap();
+
+  if ( originalMetadata.value( QStringLiteral( "creation_year" ) ).toInt() > 0 && originalMetadata.contains( QStringLiteral( "creation_doy" ) ) )
+  {
+    QDate creationDate( originalMetadata.value( QStringLiteral( "creation_year" ) ).toInt(), 1, 1 );
+    creationDate = creationDate.addDays( originalMetadata.value( QStringLiteral( "creation_doy" ) ).toInt() );
+
+    myMetadata += QStringLiteral( "<tr><td class=\"highlight\">" )
+                  + tr( "Creation date" ) + QStringLiteral( "</td><td>" )
+                  + creationDate.toString( Qt::ISODate )
+                  + QStringLiteral( "</td></tr>\n" );
+  }
+  if ( originalMetadata.contains( QStringLiteral( "major_version" ) ) && originalMetadata.contains( QStringLiteral( "minor_version" ) ) )
+  {
+    myMetadata += QStringLiteral( "<tr><td class=\"highlight\">" )
+                  + tr( "Version" ) + QStringLiteral( "</td><td>" )
+                  + QStringLiteral( "%1.%2" ).arg( originalMetadata.value( QStringLiteral( "major_version" ) ).toString(),
+                      originalMetadata.value( QStringLiteral( "minor_version" ) ).toString() )
+                  + QStringLiteral( "</td></tr>\n" );
+  }
```

Comment:
For LAS/LAZ files there are few more interesting bits of information:
- dataformat_id - which "Point Data Record Format" is used internally (that also implies which attributes are present)
- scale_x, scale_y, scale_z - what is the precision of coordinates (e.g. 0.01)
- offset_x, offset_y, offset_z - could be interesting, but probably mostly stuff for geeks

Reply:
Thanks @wonder-sk, I'll add these!

For very useful, common metadata attributes we may want to consider also adding explicit getters to the data provider class, but we can do that when we identify the need for each...